app/scripts/modules/amazon/src/serverGroup/configure/serverGroupConfiguration.service.ts

```diff
@@ -122,15 +122,20 @@ export class AwsServerGroupConfigurationService {
     'Default',
   ];
 
+  public static $inject = [
```

Comment:
Did we shut off the lint rule that complains about statics not being first?

Reply:
That was rhetorical, it's called `member-ordering` and we did.

Reply:
ye, finally.  what a horrible rule 4107b1d4f7fc1cfdefa24801a1b14ae5be565537